### PR TITLE
doc: update ReadMe file with the appropriate info.

### PR DESCRIPTION
**Pull request by mdey1**
## Proposed changes

Currently our ReadMe only mentions the ability to create search pages, which is not the case anymore. I've updated the info to highlight the key features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 ## About
 
-Coveo CLI is a command line interface to interact with the Coveo platform. It allows you to quickly create a [Coveo Headless](https://docs.coveo.com/headless)-powered search page for your Coveo organization, relying on [Angular](https://angular.io), [React](https://reactjs.org/) or [Vue.js](https://vuejs.org/).
+Coveo CLI is a command line interface to interact with the Coveo platform. With this tool, you are able to create [Coveo Headless](https://docs.coveo.com/headless)-powered search pages, push data to your organization, automate resource deployments, and create your own test organizations.
```
Comment (louis-bompart) [inline README.md:11]: For now I think it's nice, prolly outta find a more generic thingy later down the road; pretty sure we'll forget to keep that updated :D

**Review by louis-bompart — APPROVED** [review]

**Comment by louis-bompart** [comment]
iirc this is reflected in the doc website, so invoking @jpmarceau 🧙

**Comment by jpmarceau** [comment]
This overlaps with @y-lakhdar 's PR here: https://github.com/coveo/cli/pull/561, which seems ready to merge 🙃

**Comment by olamothe** [comment]
> This overlaps with @y-lakhdar 's PR here: #561, which seems ready to merge 🙃

I merged Yassine's PR @mdey1 , you will be able to pull and merge (and fix conflicts ;) )

**Review by olamothe — APPROVED** [review]

**Comment by mdey1** [comment]
> > This overlaps with @y-lakhdar 's PR here: #561, which seems ready to merge 🙃
> 
> I merged Yassine's PR @mdey1 , you will be able to pull and merge (and fix conflicts ;) )

Oh oupss! I didn't notice his PR, that's my bad. His PR seems to have all the necessary information and it's been reviewed by JP as well, so I don't think mine is necessary then :) I'll close this PR.

**Pull request closed without merging**